plugin-dev/Source/Sentry/Private/SentryEvent.cpp

```diff
@@ -26,6 +26,23 @@ USentryEvent::USentryEvent()
 	}
 }
 
+USentryEvent* USentryEvent::CreateEmptyEvent()
+{
+	return NewObject<USentryEvent>();
+}
+
+USentryEvent* USentryEvent::CreateEventWithMessageAndLevel(const FString& Message, ESentryLevel Level)
+{
+	USentryEvent* Event = NewObject<USentryEvent>();
+
+	if(!Message.IsEmpty())
+		Event->SetMessage(Message);
```

Comment:
```suggestion
	if(!Message.IsEmpty())
	{
		Event->SetMessage(Message);
	}
```
Just to stay consistent with the rest.